packages/models-library/src/models_library/services_resources.py

```diff
@@ -22,4 +48,128 @@ def ensure_limits_are_equal_or_above_reservations(cls, values):
         return values
 
 
-ServiceResources = DictModel[ResourceName, ResourceValue]
+ResourcesDict = dict[ResourceName, ResourceValue]
+
+
+class ImageResources(BaseModel):
+    image: ComposeImage = Field(
```

Comment:
Here I still disagree. Can we not use the proper name please? what means ComposeImage? this is a Docker image name, I do not know what is a compose image name. that would be the name of the docker-compose file?

Reply:
Yes, sounds reasonable.